backend/src/app.ts

```diff
@@ -62,6 +63,13 @@ app.use(express.urlencoded({ extended: true })); // for parsing application/x-ww
 dotenv.config();
 app.use(cors({ origin: process.env.CLIENT }));
 
+let mailerService: MailerService;
+if (process.env.NODE_ENV === 'test') {
+  mailerService = new MailerServiceFake();
```

Comment:
is this solving the mail limit issue on the github checks?

Reply:
Yes, we are executing e2e tests with in package.json:

`"e2e": "NODE_ENV=test jest --testPathPattern=src/tests --runInBand",`